### PR TITLE
Update LICENSE 2

### PR DESCRIPTION
**Pull request by drabiter**
license bo.

### CODE REVIEW

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

File: LICENSE [hunk 1]
```diff
@@ -20,3 +20,5 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
+xxxxxxx
+yyyyyyyA
```
Comment (drabiter) [inline LICENSE]: this sucks.

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

File: LICENSE [hunk 2]
```diff
@@ -20,3 +20,5 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
+xxxxxxx
+yyyyyyyB
```
Comment (drabiter) [inline LICENSE]: Please not B

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

File: LICENSE [hunk 3]
```diff
@@ -20,3 +20,5 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
+xxxxxxx
+yyyyyyyR
```
Comment (drabiter) [inline LICENSE]: A-E only please.

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by drabiter** [comment]
:ok_hand:

**Comment by andesyudanto** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
issue comment

**Comment by gde-vt** [comment]
:ok_hand: #design

**Comment by gde-vt** [comment]
:ok_hand: #design

**Comment by gde-vt** [comment]
:ok_hand: #bugfix

**Comment by drabiter** [comment]
:ok_hand: #design

**Comment by drabiter** [comment]
:ok

**Comment by drabiter** [comment]
:ok_hand: #design

**Comment by drabiter** [comment]
:ok_hand: #design

**Comment by drabiter** [comment]
:ok_hand: #design

**Comment by drabiter** [comment]
:ok_hand: #design

**Comment by drabiter** [comment]
:ok_hand: #bugfix

**Pull request closed without merging**